Documentation rewrite, stage 1 (#14)
delfick/photons-core: docs/start/configuration/index.rst

        before:
          - filename: "/path/to/file/you/want/loaded/before/this/one.yml

All configuration files are automatically merged to create a single
Python dictionary using `Option Merge <https://delfick-project.readthedocs.io/en/latest/api/option_merge/index.html>`_.

The following options can be configured.

Logs colours
------------

Change the color scheme used by the console log by setting the ``term_colors``
variable. This is best configured as a per-user setting in ``~/.photons_apprc.yml``

.. code-block:: yaml

    ---

    term_colors: light

> .. note:: log theme is set when the configuration is loaded when Photons starts.

Reviewer: ```suggestion
.. note:: log theme is set when the configuration is loaded after Photons starts.
```

        options:
          default_broadcast: 192.168.1.255

> The target can also be renamed:

Reviewer: it's more another target rather than a renamed target. By adding your own, the default `lan` target still exists

        options:
          default_broadcast: 192.168.1.255

> If a custom target is configured, it replaces the ``lan`` target when using

Reviewer: ```suggestion
If a custom target is configured, it can be used instead of the ``lan`` target
```
